packages/api-client/src/conversation/ConversationAPI/ConversationAPI.ts

```diff
@@ -104,7 +104,7 @@ export class ConversationAPI {
   constructor(protected readonly client: HttpClient, protected readonly backendFeatures: BackendFeatures) {}
 
   private generateBaseConversationUrl(conversationId: QualifiedId): string {
-    return this.backendFeatures.federationEndpoints
+    return this.backendFeatures.federationEndpoints && conversationId.domain
       ? `${ConversationAPI.URL.CONVERSATIONS}/${conversationId.domain}/${conversationId.id}`
       : `${ConversationAPI.URL.CONVERSATIONS}/${conversationId.id}`;
```

Comment:
this way we're completely sure that in case of empty string for `domain` we won't get url like `${ConversationAPI.URL.CONVERSATIONS}/${''}/${conversationId.id}` => `/conversations}//conversation-id`